election-portal/app/election-breakdown/modules/EBMap.tsx

```diff
@@ -637,47 +637,121 @@ const EBMap: React.FC = () => {
   const raceType = sharedState.breakdown;
   const year = sharedState.year;
 
+  const [chart, setChart] = useState<any>(null);
   const [geoData, setGeoData] = useState<any>(null);
 
+  const [wasPanned, setWasPanned] = useState(false);
+
+  const startPos = useRef<{ x: number; y: number } | null>(null);
+
+  const handleMouseDown = (event: MouseEvent) => {
+    startPos.current = { x: event.clientX, y: event.clientY };
+  };
+
+  const handleMouseUp = (event: MouseEvent) => {
+    if (startPos.current) {
+      const deltaX = Math.abs(event.clientX - startPos.current.x);
+      const deltaY = Math.abs(event.clientY - startPos.current.y);
+      if (deltaX > 10 || deltaY > 10) {
+        setWasPanned(true);
+      } else {
+        setWasPanned(false);
+      }
+      startPos.current = null;
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener("mousedown", handleMouseDown);
+    document.addEventListener("mouseup", handleMouseUp);
+
+    return () => {
+      document.removeEventListener("mousedown", handleMouseDown);
+      document.removeEventListener("mouseup", handleMouseUp);
+    };
+  }, []);
+
   const fetchMapDataAndInitializeMap = async () => {
     console.log("Map data has begun to load")
     const geoResponse = await fetch(
       "https://code.highcharts.com/mapdata/countries/us/us-all.geo.json"
     );
     let geoData = await geoResponse.json();
     console.log("Map data loaded")
+    console.log(geoData);
```

Comment:
These should either be removed now or an issue needs to be made referencing this line number so we don't ship the console log